extensions/wikia/WikiFactory/DeadWikis/maintenance.php

```diff
@@ -382,6 +398,74 @@ private function disableDiscussion( $cityId ) {
 		}
 	}
 
+	// Gets the most recent post from the specified site and returns whether it was made less than 60 days ago
+	private function isActiveSite($site_id)
+	{
+		$apiClient = $this->getSitesApi()->getApiClient();
+
+		$resourcePathTemplate = "/{siteId}/posts";
+		$httpBody = '';
+		$queryParams = array();
+		$headerParams = array();
+
+		// header params
+		$_header_accept = $apiClient->selectHeaderAccept(array('application/hal+json'));
+		if (!is_null($_header_accept)) {
+			$headerParams['Accept'] = $_header_accept;
+		}
+		$headerParams['Content-Type'] = $apiClient->selectHeaderContentType(array('application/json'));
+
+		// path params
+		$resourcePath = str_replace(
+				"{siteId}",
+				$apiClient->getSerializer()->toPathValue($site_id),
+				$resourcePathTemplate
+		);
+
+		// only need most recent post
+		$queryParams['limit'] = 1;
+
+
+		$response = null;
+
+		// make the API Call
+		try {
+			list($rawResponse, $statusCode, $httpHeader) = $apiClient->callApi(
+					$resourcePath,
+					'GET',
+					$queryParams,
+					$httpBody,
+					$headerParams,
+					'object',
+					$resourcePathTemplate
+			);
+			$response = $apiClient->getSerializer()->deserialize($rawResponse, 'object', $httpHeader);
+		} catch (\Swagger\Client\ApiException $e) {
+			switch ($e->getCode()) {
+				case 204:
+					$data = $apiClient->getSerializer()->deserialize($e->getResponseBody(), 'object', $e->getResponseHeaders());
```

Comment:
Several of these lines are over 120 characters (https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#line-length).  Its likely if you follow Ciacho's suggestion of splitting this method into smaller methods, many of these will be fixed.